addons/web/migrations/12.0.1.0/pre-migration.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2019 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from psycopg2.extensions import AsIs
+from openupgradelib import openupgrade
+
+
+def remove_web_planner_constraints(env):
+    """Remove foreign constraints on merged `web_planner` module for avoiding
+    errors when removing referenced records.
+    """
+    constraint_list =  [
```

Comment:
flake8

Reply:
Fixed. Please squash on merge. I remember someone talking about this problem in one PR, but I'm not able to find the thread.